src/main/java/com/google/sps/data/Reaction.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/** 
+ * Reactions a user can add to a post 
+ */
```

Comment:
see a better example of an enum

https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html